cloudinit/config/cc_mounts.py

```diff
@@ -248,8 +248,11 @@ def create_swap(fname, size, method):
     util.ensure_dir(swap_dir)
 
     fstype = util.get_mount_info(swap_dir)[1]
+    kernel_version = re.sub(r'.*?(\d+(\.\d+)?).*', r'\1', str(os.uname()[2]))
```

Comment:
`os.uname()` returns a named tuple of strings, so we can simplify/clarify this a touch:

```suggestion
    kernel_version = re.sub(r'.*?(\d+(\.\d+)?).*', r'\1', os.uname().release)
```